packages/ui-tests/cypress/fixtures/flows/SqlBeansCR.yaml

```diff
@@ -0,0 +1,26 @@
+- route:
+    id: SqlBeansRoute
+    from:
+      uri: timer
+      steps:
+        - to:
+            uri: sql
+        - to:
+            uri: log:test
+- beans:
+    - name: mysqlDataSource
+      type: io.kaoto.mysqlDataSource
```

Comment:
Trivial but usually the class name starts with a capital letter, so `MysqlDataSource` or `MySQLDataSource` might look nice

Reply:
thanks for noticing! fixed